packages/elements/source/composite/alert/page.ts

```diff
@@ -1,156 +1,103 @@
-const ELEMENT_ALERT_PAGE_ICON = 'alert-page-icon';
-
-const OVERWRITE_THEME_DARK_CONTAINER = `.${ELEMENT_ALERT_PAGE_CONTAINER}${IS_THEME_DARK}`;
-const OVERWRITE_THEME_DARK_CLOSE_BUTTON = `${OVERWRITE_THEME_DARK_CONTAINER} .${ELEMENT_ALERT_PAGE_CLOSE_BUTTON}`;
-
-const OverwriteThemeStyles = `
-  ${OVERWRITE_THEME_DARK_CONTAINER} * {
-    color: ${token.color.white};
-  }
-
-  ${OVERWRITE_THEME_DARK_CLOSE_BUTTON} rect {
-    fill: ${token.color.white};
-  }
-`;
-
-const ButtonStyles = `
-  .${ELEMENT_ALERT_PAGE_CLOSE_BUTTON} {
-    position: absolute;
-    top: ${token.spacing.lg};
-    right: ${token.spacing.lg};
-  }
-
-  .${ELEMENT_ALERT_PAGE_CLOSE_BUTTON} rect {
-    fill: ${token.color.black};
-  }
-
-  @container ${ELEMENT_NAME} (max-width: ${MEDUM}px) {
-    .${ELEMENT_ALERT_PAGE_CLOSE_BUTTON} {
-      top: ${token.spacing.sm};
-      right: ${token.spacing.sm};
-    }
-  }
-`;
-
-const IconStyles = `
-  .${ELEMENT_ALERT_PAGE_ICON} {
-    display: block;
-    fill: ${token.color.gold};
-  }
-
-  @container ${ELEMENT_NAME} (max-width: ${MEDUM}px) {
-    .${ELEMENT_ALERT_PAGE_ICON} {
-      position: absolute;
-      top: -20px;
-    }
-  }
-`;
-
-const STYLES_ALERT_PAGE_ELEMENT = `
-  .${ELEMENT_ALERT_PAGE_DECLARATION} {
-    container: ${ELEMENT_NAME} / inline-size;
-  }
-
-  .${ELEMENT_ALERT_PAGE_CONTAINER} {
-    display: flex;
-    position: relative;
-    padding: ${token.spacing.lg};
-    padding-right: ${token.spacing['2xl']};
-    gap: ${token.spacing.lg};
-    border: solid 4px ${token.color.gold};
-  }
-
-  @container ${ELEMENT_NAME} (max-width: ${MEDUM}px) {
-    .${ELEMENT_ALERT_PAGE_CONTAINER} {
-      padding-right: ${token.spacing.lg};
-    }
-  }
-
-  ${IconStyles}
-  ${ButtonStyles}
-  ${OverwriteThemeStyles}
-`;
-
-const CreateIcon = () => {
-  const iconWrapper = document.createElement('div');
-  let icon = iconWarning;
-
-  iconWrapper.classList.add(ELEMENT_ALERT_PAGE_ICON);
-  iconWrapper.innerHTML = icon;
+const createContainer = (
+  props: Pick<
+    AlertPageProps,
+    | 'isShowIcon'
+    | 'isThemeDark'
+    | 'isThemeLight'
+    | 'headline'
+    | 'text'
+    | 'actions'
+  >,
+): ElementVisual => {
+  const { isShowIcon = true, isThemeDark } = props;
+
+  const textModel = AlertText(props);
+  const iconModel = isShowIcon ? createIcon(iconWarning) : null;
+
+  const closeButtonModel = CreateCloseButton({
+    isThemeDark: !!isThemeDark,
+    targetSelector: '.alert-page-declaration',
+  });
 
-  return iconWrapper;
+  return ElementBuilder.create.div({
+    className: 'alert-page-container',
+    children: [iconModel, textModel, closeButtonModel].filter(
+      Boolean,
+    ) as ElementVisual[],
+    elementStyles: {
+      element: {
+        display: 'flex',
+        position: 'relative',
+        padding: token.spacing.lg,
+        paddingRight: token.spacing['2xl'],
+        gap: token.spacing.lg,
+        border: `solid 4px ${token.color.gold}`,
+
+        ['&[theme="dark"] *']: {
```

Comment:
I wonder can we use subElement for this? I'm not sure but in my imagination it should do the same thing